float64 arm1
float64 arm2
float64 arm3
float64 arm4
float64 arm5
float64 arm6